### PR TITLE
Add: Precise number of blocks that may be issued (with K=2160).

### PR DESCRIPTION
**Pull request by nc6**

### CODE REVIEW

File: specs/chain/latex/blockchain-spec.tex [hunk 1]
```diff
@@ -552,7 +552,9 @@ \section{Counting signed blocks}
 blocks signed by keys that $sk_s$ delegated to is no more than a threshold $K
 \cdot t$, where $t$ is a constant that will be picked in the range
 $1/5 \leq t \leq 1/4$. Initial research suggests setting $t=0.22$ as a good
-value. See Appendix \ref{apdx:calculating-t} for the background on this value.
+value. Specifically, given $K=2160$, we would allow a single genesis key to
+issue (via delegates) 475 blocks, but a 476$^{\text{th}}$ block would be
```
Comment (dnadales) [inline specs/chain/latex/blockchain-spec.tex]: Don't we want all the numbers to be rendered in math font? For instance `2160` and `475` will be rendered with slightly different fonts: the former in within a math environment, whereas the latter isn't.

**Review by dnadales — APPROVED** [review]

File: specs/chain/latex/blockchain-spec.tex (hunk 1 above)
Comment (nc6) [reply specs/chain/latex/blockchain-spec.tex]: Yep, good point...

Comment (nc6) [inline specs/chain/latex/blockchain-spec.tex]:
```suggestion
issue (via delegates) $475$ blocks, but a $476${\text{th}}$ block would be
```

**Review by nc6 — COMMENTED** [review]
Whoops

File: specs/chain/latex/blockchain-spec.tex [hunk 2]
```diff
@@ -552,7 +552,9 @@ \section{Counting signed blocks}
 blocks signed by keys that $sk_s$ delegated to is no more than a threshold $K
 \cdot t$, where $t$ is a constant that will be picked in the range
 $1/5 \leq t \leq 1/4$. Initial research suggests setting $t=0.22$ as a good
-value. See Appendix \ref{apdx:calculating-t} for the background on this value.
+value. Specifically, given $K=2160$, we would allow a single genesis key to
+issue (via delegates) $475$ blocks, but a $476${\text{th}}$ block would be
```
Comment (dnadales) [inline specs/chain/latex/blockchain-spec.tex]: You have three `$` (dollars :P) here :D

Comment (nc6) [reply specs/chain/latex/blockchain-spec.tex]: rgh, I thought I removed that one..

Comment (nc6) [inline specs/chain/latex/blockchain-spec.tex]:
```suggestion
issue (via delegates) $475$ blocks, but a $476^{\text{th}}$ block would be
```

**Comment by nc6** [comment]
bors retry

**Pull request merged**